ui/src/table.tsx

```diff
@@ -486,16 +490,33 @@ const
           const epoch = Number(v)
           v = new Date(isNaN(epoch) ? v : epoch).toLocaleString()
         }
+
         if (col.key === primaryColumnKey) {
           const onClick = () => {
             wave.args[m.name] = [item.key as S]
             wave.push()
           }
-          return <Fluent.Link onClick={onClick}><TooltipWrapper>{v}</TooltipWrapper></Fluent.Link>
+          const align = col.align || 'left'
+          return (
+            <Fluent.Link onClick={onClick}>
+              <div style={{ display: 'flex', justifyContent: align, textAlign: align }}>
+                <TooltipWrapper>{v}</TooltipWrapper>
+              </div>
+            </Fluent.Link>)
         }
 
         return <TooltipWrapper>{v}</TooltipWrapper>
       },
+      onRenderItemColumn = (item?: Fluent.IObjectWithKey & Dict<any>, _idx?: U, col?: WaveColumn) => {
+        if (!item || !col) return <span />
+
+        const AlignmentWrapper = ({ children }: { children: any }) => {
+          const align = col.align || 'left'
+          return <div style={{ display: 'flex', justifyContent: align, textAlign: align }}>{children}</div>
+        }
+
+        return <AlignmentWrapper>{getItemColumnStyle(item, _idx, col)}</AlignmentWrapper>
```

Comment:
Almost there, no need for separate component if it's used just once anyway. The simpler the code the better.

```suggestion
        const align = col.align || 'left'
        return <div style={{ display: 'flex', justifyContent: align, textAlign: align }}>{getItemColumnStyle(item, _idx, col)}</div>
```